tests/marqo_test.py

```diff
@@ -180,24 +199,31 @@ def warm_request(self, func, *args, **kwargs):
             func(*args, **kwargs)
 
     def create_cloud_index(self, index_name, settings_dict=None, **kwargs):
+        """
+        Creates a cloud index with the given name and settings. If settings_dict or any kwargs are provided,
+        they will be used to create a unique index name by hashing the settings_dict and kwargs.
+        If settings_dict is not provided, the index name will be the given index_name + the index_suffix.
```

Comment:
Make sure the language is accurate:

`If settings_dict is not provided,` -> `If settings_dict and index settings kwargs are not provided...`